spec/rubocop/cop/performance/redundant_match_spec.rb

```diff
@@ -111,4 +111,15 @@ def method(str)
                    ^^^^^^^^^^^^^^^^^^ Use `=~` in places where the `MatchData` returned by `#match` will not be used.
     RUBY
   end
+
+  it 'autocorrects Regexp#match to Regexp#=~' do
```

Comment:
```suggestion
  it 'registers an offense when receiver is a Regexp literal' do
```

Reply:
I think it's a good to change about minor wording, but doesn't this completely lose the nuance of autocorrect support?
Or should we prefer that the message be ignorant as to whether autocorrect is supported or not?

Reply:
`it 'registers an offense and corrects when receiver is a Regexp literal' do`, WDYT?

Reply:
It sounds good and much clear 👍